src/directory.build.props

```diff
@@ -30,7 +31,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition="'$(IsTestProject)' != 'true'">
-    <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.SourceLink.GitHub" IncludeAssets="analyzers;build;buildMultitargeting;buildTransitive" PrivateAssets="All" />
```

Comment:
Don't think we need the IncludeAssets to be honest with you, PrivateAssets is the important one.

Reply:
it was duplicated, so kept the unusual one, I will update now